admin/future_grants/V1.6.0__remove_erroneous_grant.sql

```diff
@@ -48,12 +48,13 @@ REVOKE OWNERSHIP
 REVOKE READ, USAGE
 	ON FUTURE STAGES
 	IN SCHEMA SYNAPSE_DATA_WAREHOUSE_DEV.SYNAPSE
-	FROM DATABASE ROLE SYNAPSE_DATA_WAREHOUSE_DEV.SYNAPSE_STAGE_READ
+	FROM DATABASE ROLE SYNAPSE_DATA_WAREHOUSE_DEV.SYNAPSE_STAGE_READ;
```

Comment:
It looks like commits unrelated to your work are now showing up in the diff :/ Is this what Phil was eluding to in the discussion from https://github.com/Sage-Bionetworks/snowflake/pull/138#issuecomment-2654738545 ?

Reply:
Oops, just saw this, yes this is related to that discussion. Some duplicate commits from `dev` were dropped as Phil was cleaning up the branches, and it's showing up as a new commit being introduced to `dev` from here. I made a new branch just to start from a clean slate.